tests/app/test_queue.py

```diff
@@ -75,33 +74,33 @@ def delete_all_inflight(self, redis):
             redis.delete(key)
 
     @pytest.mark.serial
-    def test_put_mesages(self, redis, redis_queue):
+    def test_put_mesages(self, app, redis, redis_queue):
         element = generate_element()
         redis_queue.publish(element)
-        assert redis.llen(Buffer.INBOX.name(QNAME_SUFFIX)) == 1
+        assert redis.llen(Buffer.INBOX.name(app.config["NOTIFY_ENVIRONMENT"])) == 1
         self.delete_all_list(redis)
 
     @pytest.mark.serial
-    def test_polling_message(self, redis, redis_queue):
+    def test_polling_message(self, app, redis, redis_queue):
```

Comment:
This is not necessary to define the `app` mock as a parameter because of the `autouse=True` usage [at this line](https://github.com/cds-snc/notification-api/pull/1451/files#diff-c9d36498371981109d1bf2de7b12bb84964b6f4a6b55a03af7bf94ca02781366R22).

Reply:
An example would be helpful.